demisto_sdk/commands/test_content/tests/test_context_test.py

```diff
@@ -262,3 +265,207 @@ def test_replacing_placeholders(mocker, tmp_path):
     integration = Integration(build_context, 'integration_with_placeholders', ['instance'])
     integration._set_integration_params(server_url='1.2.3.4', playbook_id='playbook_integration', is_mockable=False)
     assert '%%SERVER_HOST%%' in build_context.secret_conf.integrations[0].params.get('url')
+
+
+CASES = [
+    (  # case one input is found
+        {'id': 'pb_test',
+         'inputs': [{'key': 'Endpoint_hostname', 'value': {'simple': '', 'complex': None}, 'required': False,
+                     'description': 'The hostname of the endpoint to isolate.', 'playbookInputQuery': None},
+                    {'key': 'ManualHunting.DetectedHosts', 'value': {'simple': '', 'complex': None}, 'required': False,
+                     'description': 'Hosts that were detected as infected during the manual hunting.',
+                     'playbookInputQuery': None},
+                    {'key': 'Endpoint_ip', 'value': {'simple': '', 'complex': None}, 'required': False,
+                     'description': 'The IP of the endpoint to isolate.', 'playbookInputQuery': None},
+                    {'key': 'Endpoint_id', 'value': {'simple': '', 'complex': None}, 'required': False,
+                     'description': 'The ID of the endpoint to isolate.', 'playbookInputQuery': None}],
+         },
+        {
+            "playbookID": "pb_test",
+            "input_parameters": {
+                "Endpoint_hostname": {
+                    "simple": "test"
+                },
+            }
+        },
+        [{'key': 'Endpoint_hostname', 'value': {'simple': 'test', 'complex': None}, 'required': False,
+          'description': 'The hostname of the endpoint to isolate.', 'playbookInputQuery': None},
+         {'key': 'ManualHunting.DetectedHosts', 'value': {'simple': '', 'complex': None}, 'required': False,
+          'description': 'Hosts that were detected as infected during the manual hunting.',
+          'playbookInputQuery': None}, {'key': 'Endpoint_ip', 'value': {'simple': '', 'complex': None},
+                                        'required': False, 'description': 'The IP of the endpoint to isolate.',
+                                        'playbookInputQuery': None}, {'key': 'Endpoint_id',
+                                                                      'value': {'simple': '', 'complex': None},
+                                                                      'required': False,
+                                                                      'description': 'The ID of the endpoint to isolate.',
+                                                                      'playbookInputQuery': None}]
+
+    ),
+]
+
+
+@pytest.mark.parametrize('current, new_configuration, expected', CASES)
+def test_replacing_pb_inputs(mocker, current, new_configuration, expected):
```

Comment:
Please add Given-When-Then docstring to the test (this comment goes for all the tests below).